README.md

```diff
@@ -89,7 +89,7 @@ Tests are run with the top-level `Makefile`:
 make test
 ```
 
-It is also possible to run then as 
+It is also possible to run then as
 
 ```bash
 python setup.py test
```

Comment:
The Contribution guidelines should be replaced with some inline text rather than referencing a document that is not external facing.